tests/unit/test_charm.py

```diff
@@ -10,7 +10,7 @@
 )
 from uuid import uuid4
 from ops.testing import Harness
-from charm import AptMirrorCharm
+from src.charm import AptMirrorCharm
```

Comment:
I think we should use `from charm import AptMirrorCharm` and define proper `PYTHONPATH` in `tox.ini`.

Reply:
Done